src/test/scala/org/ergoplatform/nodeView/history/NonVerifyADHistorySpecification.scala

```diff
@@ -16,15 +16,6 @@ class NonVerifyADHistorySpecification extends HistoryTestHelpers {
 
   private lazy val popowHistory = ensureMinimalHeight(genHistory(), 100)
 
-  ignore("Should apply UTXOSnapshotChunks") {
```

Comment:
I think such a test, that applies `UtxoSnapshotManifest` and `UtxoSnapshotChunk` is needed

Reply:
@catena2w, tests like this one are moved to a more suitable place.

Reply:
Where is it? I don't see any tests, that apply chunks to history